charts/victoria-metrics-gateway/templates/service-monitor.yaml

```diff
@@ -0,0 +1,40 @@
+{{- if .Values.serviceMonitor.enabled -}}
+apiVersion: monitoring.coreos.com/v1
```

Comment:
should we use native VM resource here? i mean service scrape?

Reply:
just to be compatible with other charts